capstone/src/main/resources/templates/event.html

```diff
@@ -1,28 +1,19 @@
 <!DOCTYPE html>
-<html>
+<html xmlns:th="http://www.thymeleaf.org">
 
   <head>
     <meta charset="UTF-8">
     <title> Create an Event </title>
     <script src="event.js"> </script>
+    <script src="user.js"> </script>
+    <script src="script.js"> </script>
     <link rel="stylesheet" href="style.css">
   </head>
-  <body onload="loadEventInfo();">
-
-    <nav class="nav-bar">
-        <ul>
-            <li class="home"><a href="index.html">College Activity Finder</a></li>
-            <li class="active"><a href="index.html">Events</a></li>
-            <li><a href="organizationsearch.html">Organizations</a></li>
-            <li class=""><a href="profile.html">My Account</a></li>
-            <li class=""><a href="map.html">Event Map</a> </li>
-            <li><a href="=">About Us</a></li>
-        </ul>
-    </nav>
-
-    <h1>Create an Event </h1>
-
-    <div class="event-form">
+  <body onload="getUser(false, 'general-profile', 'org-event');loadEventInfo();">
+    <div th:insert="fragments/navbar::navbar"></div>
```

Comment:
This creates an extra parent `div` around the `nav`. I don't think it changes anything visually, but if you wanted to avoid the `div`, I think you can use `<nav th:replace="fragments/navbar::navbar">`.